.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Any chance we can avoid this file? I've never needed it before.

Reply:
yeah I guess. a newer version of watchman will error without this file, but that's only internal now. will remove.

Reply:
Can you talk to those wonderful internal people and say that I will be quite sad if this is the case in the future 😅

Reply:
yeah I think they're aware. I'll check.